libs/api/domains/user-profile/src/lib/userProfile.service.ts

```diff
@@ -38,6 +42,41 @@ export class UserProfileService {
     return this.userProfileApi.withMiddleware(new AuthMiddleware(auth))
   }
 
+  async getIslykillProfile(user: User) {
+    try {
+      const islyklarData = await this.islyklarService.getIslykillSettings(
+        user.nationalId,
+      )
+
+      const feature = await this.featureFlagService.getValue(
+        Features.personalInformation,
+        false,
+        user,
+      )
+      if (!feature) {
+        return null
+      }
+
+      return {
+        nationalId: user.nationalId,
+        emailVerified: false,
+        mobilePhoneNumberVerified: false,
+        documentNotifications: false,
+        emailStatus: 'NOT_VERIFIED',
+        mobileStatus: 'NOT_VERIFIED',
+
+        // Islyklar data:
+        mobilePhoneNumber: islyklarData?.mobile,
+        email: islyklarData?.email,
+        canNudge: islyklarData?.canNudge,
```

Comment:
`canNudge` was removed from the islykill.service in this pull request. Is this for backwards compatibility or something else?

Reply:
Though I am seeing at as optional in some inputs elsewhere, is this not being returned?